### PR TITLE
Yixuan and WeiPin shopping list

### PR DESCRIPTION
**Pull request by jyxhazcake**
Done:
- Overall layout of the page (where components are placed)
- Add item (item name and quantity) to shopping list
- Increase/ Decrease Quantity of items
- Styling (for one component lol)

Screenshot:
image

### CODE REVIEW

File: src/pages/Shopping_YXWP/index.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    {
+      key: 2,
+      itemQuantity: 4,
+      itemName: 'Dummy Item 2',
+    },
+  ])
+
+  const handleAddItem = () => {
+    setData((prevData: Props[]) => [...prevData, { key: index, itemQuantity: quantity, itemName: name }])
+    setIndex(index + 1)
```
Comment (rldefa) [inline src/pages/Shopping_YXWP/index.tsx:32]: nice

File: src/pages/Shopping_YXWP/index.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  }
+
+  const handleIncreaseQuantity = (currItem: Props) => {
+    setData(data.map((item) => (item.key === currItem.key ? { ...item, itemQuantity: item.itemQuantity + 1 } : item)))
+  }
```
Comment (rldefa) [inline src/pages/Shopping_YXWP/index.tsx:37]: Well implemented, good function name too

**Review by rldefa — APPROVED** [review]
lgtm 👍

File: src/pages/Shopping_YXWP/index.tsx (hunk 2 above)
Comment (jyxhazcake) [reply src/pages/Shopping_YXWP/index.tsx:37]: ty

File: src/pages/Shopping_YXWP/index.tsx (hunk 1 above)
Comment (jyxhazcake) [reply src/pages/Shopping_YXWP/index.tsx:32]: ur nicer :)

**Pull request merged**